src/ucp/core/ucp_context.c

```diff
@@ -67,7 +67,7 @@ static ucs_config_field_t ucp_config_table[] = {
    "Threshold for switching from short to bcopy protocol",
    ucs_offsetof(ucp_config_t, ctx.bcopy_thresh), UCS_CONFIG_TYPE_MEMUNITS},
 
-  {"RNDV_THRESH", "1gb",
+  {"RNDV_THRESH", "1mb",
```

Comment:
Right. It should be detected by ucx according to the capabilities that a transport exposes, like the zcopy threshold for example. This will be added in a future PR.